lib/data/external/GCP/GcpApis/deleteObjects.js

```diff
@@ -0,0 +1,96 @@
+const async = require('async');
+const request = require('request');
+const uuid = require('uuid/v4');
+const { errors } = require('arsenal');
+
+const { logger, JsonError } = require('../GcpUtils');
+const { logHelper } = require('../../utils');
+
+function formBatchRequest(bucket, deleteList) {
+    let retBody = '';
+    const boundary = uuid().replace(/-/g, '');
+
+    deleteList.forEach(object => {
+        // add boundary
+        retBody += `--${boundary}\n`;
```

Comment:
You should replace `\n` with `\r\n` in the HTTP forged header to be compliant with HTTP spec (otherwise it's not guaranteed to be accepted by all servers).